data/ecosystem/vendors.yaml

```diff
@@ -41,6 +41,13 @@
   contact: 'hello@betterstack.com'
   oss: false
   commercial: true
+- name: Causely
+  distribution: false
+  nativeOTLP: true
+  url: 'https://www.causely.io/blog/unveiling-the-causal-revolution-in-observability/'
+  contact: 'early-access@causely.io'
```

Comment:
this contact is for us to follow up with you if we see any issues like a broken link, a github handle is preferred, an email address is OK as well.

Reply:
changed it to our support email